doc_src/cmds/fish_command_not_found.rst

```diff
@@ -0,0 +1,77 @@
+.. _cmd-fish_cmd_not_found:
+
+fish_command_not_found - what to do when a command wasn't found
+===============================================================
+
+Synopsis
+--------
+
+::
+
+  function fish_command_not_found
+      ...
+  end
+
+
+Description
+-----------
+
+When fish can't find a command, it invokes this function.
```

Comment:
Maybe `When fish can't find the command you're trying to execute, it invokes this function.`